dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -1,6 +1,7 @@
 -# TODO(JL): conditionally load on client
 - if browser.chrome? && browser.version.to_s.to_i == 34
-  %script{src: '/shared/js/chrome34Fix.js'}
+  :javascript
+    window.dashboard.isChrome34 = true;
```

Comment:
One downside is that this JS file will now ship to all clients instead of just Chrome 34.  But it's not a very big file so not a big deal.